core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -234,6 +234,20 @@ private[data] sealed abstract class OptionTInstances extends OptionTInstances0 {
       def defer[A](fa: => OptionT[F, A]): OptionT[F, A] =
         OptionT(F.defer(fa.value))
     }
+
+  implicit def optionTFunctorFilter[F[_]: Functor]: FunctorFilter[OptionT[F, ?]] = {
```

Comment:
There was a `TraverseFilter` instance for `OptionT` in #1225. Is there an issue with it?

Reply:
No idea, just took what was there from `cats-mtl`

Reply:
I'm having trouble tracking down the history in cats-mtl, but I think that we should have the `OptionT` instance. It should be have identically to the corresponding `Nested` instance.